src/index.js

```diff
@@ -31,12 +31,15 @@ export const draw = (config, xScale) =>
             .data(filteredData)
             .call(dropLine(config, xScale))
             .call(bounds(config, xScale))
-            .call(axis(config, xScale));
+            .call(axis(d3, config, xScale));
     };
 
-export default (customConfiguration = {}) =>
+export default (customConfiguration = {}, d3 = window.d3) =>
     selection => {
-        const config = defaultsDeep(customConfiguration, defaultConfiguration);
+        const config = defaultsDeep(
+            customConfiguration,
```

Comment:
Won't it be any issues with a custom D3 used here in comparison of the `d3` used in EventDrops?